### ISSUE TITLE
how do you force currency without using proxy?

### ISSUE THREAD

**Issue by yossielimelech** [sudheer-ranga/aliexpress-product-scraper#18]
I was wondering if there's a way to force the currency type, for example USD. Right now the prices are in my local currency.

**Comment by sudheer-ranga** [comment]
That is correct. Since this uses headless chrome, the currency is detected by IP address. I will take this as an enhancement.

**Comment by aimehai** [comment]
Is there any update?

**Comment by zolzaya** [comment]
any updates?
